static/app/views/performance/landing/widgets/widgets/lineChartListWidget.tsx

```diff
@@ -624,6 +684,30 @@ export function LineChartListWidget(props: PerformanceWidgetProps) {
                   </Fragment>
                 </RoutingContextProvider>
               );
+            case PerformanceWidgetSetting.HIGHEST_CACHE_MISS_RATE_TRANSACTIONS:
+              const cacheMissRate = listItem[fieldString];
+              const target = normalizeUrl(
+                `${CACHE_BASE_URL}/?${qs.stringify({transaction: transaction})}`
+              );
```

Comment:
```suggestion
              const target = normalizeUrl({
                pathname: CACHE_BASE_URL,
                query: {transaction: transaction},
              });
```

I think this also works and avoids the extra import to force this into a string if you want to use it